packages/assets/package.json

```diff
@@ -23,5 +23,7 @@
     "package.json",
     "LICENSE",
     "README.md"
-  ]
+  ],
+  "main": "index.js",
+  "module": "index.js"
```

Comment:
Is it ESM or why do we specify this?

Reply:
It's probably not necessary but we usually specify both options in our packages. Nothing more